miso-web/src/main/webapp/scripts/hot_dilution.js

```diff
@@ -160,8 +160,30 @@ HotTarget.dilution = {
       title: 'Create one pool from many dilutions',
       action: function(items) {
         HotUtils.warnIfConsentRevoked(items, function() {
-          window.location = window.location.origin + '/miso/library/dilution/bulk/merge?' + jQuery.param({
-            ids: items.map(Utils.array.getId).join(',')
+          var fields = [{
```

Comment:
This function (asking whether to create a new box) is repeated several times in this file and others. Please abstract to `HotUtils` (`hot.js`)